org.eclipse.winery.repository/pom.xml

```diff
@@ -159,7 +159,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>23.4-jre</version>
+            <version>30.1.1-jre</version>
```

Comment:
Is this the only place where guava is used? Othwerwise we should use a property for the version.

Reply:
<img width="1440" alt="Bildschirmfoto 2021-06-16 um 08 58 19" src="https://user-images.githubusercontent.com/2853992/122172546-2615d480-ce81-11eb-8d1c-aec55015287f.png">

Seems like it